tests/generic_config_updater/conftest.py

```diff
@@ -53,3 +55,28 @@ def cfg_facts(duthosts, rand_one_dut_hostname):
     """
     duthost = duthosts[rand_one_dut_hostname]
     return duthost.config_facts(host=duthost.hostname, source="persistent")['ansible_facts']
+
+
+@pytest.fixture(scope="module", autouse=True)
+def verify_configdb_with_empty_input(duthost):
+    """Fail immediately if empty input test failure
+
+    Args:
+        duthost: Hostname of DUT.
```

Comment:
`duthost` is not a hostname. It's an ansible host object that can be used to interact with the DUT device.

Reply:
Suggest @wen587 to fix the comment in another PR.

Reply:
Got it. Update in next PR: https://github.com/Azure/sonic-mgmt/pull/5509/files#diff-11b9a905e656f51367a09b1436bc8d7e57c59fecbd8e36e54e332c41a88a60e6R42